lib/e2ee/e2ee_common.h

```diff
@@ -204,7 +204,7 @@ class SignedOneTimeKey {
                     { "signatures"_ls,
                       QJsonObject {
                           { userId, QJsonObject { { "ed25519:"_ls % deviceId,
-                                                    QString(signature) } } } } } }
+                                                    QString::fromUtf8(signature) } } } } } }
```

Comment:
Signatures are Base64.
```suggestion
                                                    QString::fromLatin1(signature) } } } } } }
```